tensorflow/core/api_def/base_api/api_def_ParallelInterleaveDatasetV2.pbtxt

```diff
@@ -1,13 +1,93 @@
 op {
   graph_op_name: "ParallelInterleaveDatasetV2"
   visibility: HIDDEN
+  in_arg {
+    name: "input_dataset"
+    description: <<END
+Dataset that produces a stream of arguments for the function `f`.
+END
+  }
+  in_arg {
+    name: "other_arguments"
+    description: <<END
+Additional arguments to pass to `f` beyond those produced by `input_dataset`.
+Evaluated once when the dataset is instantiated.
+END
+  }
+  in_arg {
+    name: "cycle_length"
+    description: <<END
+Number of datasets (each created by applying `f` to the elements of
+`input_dataset`) among which the `ParallelInterleaveDatasetV2` will cycle in a
+round-robin fashion.
+END
+  }
+  in_arg {
+    name: "block_length"
+    description: <<END
+Number of elements at a time to produce from each interleaved invocation of a
+dataset returned by `f`.
+END
+  }
+  in_arg {
+    name: "num_parallel_calls"
+    description: <<END
+If this input is >=1, it bounds the number of datasets from which this instance
+of `ParallelInterleaveDatasetV2` will fetch simultaneously.
+The actual number of parallel calls will depend on the `cycle_length`
+parameter, the number of available CPUs, and the number of available datasets.
+
+If this input is set to -1 (i.e. the constant `tf.data.experimental.AUTOTUNE`
+in the Python API), then the dataset's kernel will choose an upper bound
+automatically.
+END
+  }
   attr {
     name: "f"
     description: <<END
 A function mapping elements of `input_dataset`, concatenated with
 `other_arguments`, to a Dataset variant that contains elements matching
 `output_types` and `output_shapes`.
+END
+  }
+  attr {
+    name: "Targuments"
+    description: <<END
+Types of the elements of `other_arguments`.
+END
+  }
+  attr {
+    name: "output_types"
+  }
+  attr {
+    name: "output_shapes"
+  }
+  attr {
+    name: "sloppy"
```

Comment:
I don't think this attribute is present in the ParallelInterleaveDatasetV2.

Reply:
Commit a34c748 removed this parameter.